pkg/errors/errors_test.go

```diff
@@ -3,20 +3,30 @@ package errors
 import (
 	"testing"
 
+	"fmt"
+
 	"github.com/stretchr/testify/assert"
 	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
-func TestAlreadyExists(t *testing.T) {
+func TestCreateArtifact(t *testing.T) {
```

Comment:
is this function rename accurate?

Reply:
oops must've been copy-pasta! Updated, thanks.